test/RequestHandlerTest.php

```diff
@@ -4,26 +4,23 @@ class RequestHandlerTest extends \PHPUnit_Framework_TestCase
 {
     protected function setUp()
     {
-        $headers = $this->getMock('Guzzle\Http\Message\Header\HeaderCollection',
-                                  array('setHeaders', 'getHeaders', 'toArray'),
+        $headers = PHPUnit_Framework_TestCase::createMock('Guzzle\Http\Message\Header\HeaderCollection',
```

Comment:
Can you explain your reasoning for this change? I realize that phpunit defines these static methods, but all the docs seem to use the instance aliases, so I believe they're the preferred pattern. At the time of writing it looks like https://phpunit.de/ is down, but docs for the most recent version of the library illustrate the pattern: https://phpunit.readthedocs.io/en/8.0/test-doubles.html#stubs.